web-client/integration-tests/adcClerkViewsSectionMessagesJourney.test.js

```diff
@@ -0,0 +1,285 @@
+    cerebralTest.docketNumber = caseDetail.docketNumber;
+  });
+
+  const message1Subject = `message 1 ${Date.now()}`;
+  const message2Subject = `message 2 ${Date.now()}`;
+  const message3Subject = `message 3 ${Date.now()}`;
+  const message4Subject = `message 4 ${Date.now()}`;
+  const message5Subject = `message 5 ${Date.now()}`;
+  const message6Subject = `message Completed 1 ${Date.now()}`;
+  const message7Subject = `message Completed 2 ${Date.now()}`;
+
+  // Send some messages to ADC user(s)
+  loginAs(cerebralTest, 'petitionsclerk@example.com');
+  createNewMessageOnCase(cerebralTest, {
+    subject: message1Subject,
+    toSection: 'adc',
+    toUserId: testAdcId,
+  });
+
+  loginAs(cerebralTest, 'docketclerk@example.com');
+  createNewMessageOnCase(cerebralTest, {
+    subject: message2Subject,
+    toSection: 'adc',
+    toUserId: testAdcId,
+  });
+
+  loginAs(cerebralTest, 'docketclerk@example.com');
+  createNewMessageOnCase(cerebralTest, {
+    subject: message3Subject,
+    toSection: 'adc',
+    toUserId: testAdcId,
+  });
+
+  loginAs(cerebralTest, 'adc@example.com');
+  createNewMessageOnCase(cerebralTest, {
+    subject: message4Subject,
+    toSection: 'petitions',
+    toUserId: petitionsClerkId,
+  });
+
+  createNewMessageOnCase(cerebralTest, {
+    subject: message5Subject,
+    toSection: 'petitions',
+    toUserId: petitionsClerkId,
+  });
+
+  it('go to section inbox', async () => {
+    await cerebralTest.runSequence('gotoMessagesSequence', {
+      box: 'inbox',
+      queue: 'section',
+    });
+  });
+
+  it('verify default sorting of section inbox createdAt sort field, ascending', async () => {
+    let afterInboxMessageCount = await getUserMessageCount(
+      cerebralTest,
+      'inbox',
+      'section',
+    );
+
+    const { messages: inboxMessages } = runCompute(formattedMessagesComputed, {
+      state: cerebralTest.getState(),
+    });
+
+    const expected = [message1Subject, message2Subject, message3Subject];
+
+    expect(afterInboxMessageCount).toEqual(
+      expected.length + beforeInboxMessageCount,
+    );
+
+    validateMessageOrdering(inboxMessages, expected);
+  });
+
+  it('go to section outbox', async () => {
+    await cerebralTest.runSequence('gotoMessagesSequence', {
+      box: 'outbox',
+      queue: 'section',
+    });
+  });
+
+  it('verify default sorting of section outbox createdAt sort field, descending', async () => {
+    let afterOutboxMessageCount = await getUserMessageCount(
+      cerebralTest,
+      'outbox',
+      'section',
+    );
+
+    const { messages: outboxMessages } = runCompute(formattedMessagesComputed, {
+      state: cerebralTest.getState(),
+    });
+
+    const expected = [message5Subject, message4Subject];
+
+    expect(afterOutboxMessageCount).toEqual(
+      expected.length + beforeOutboxMessageCount,
+    );
+
+    validateMessageOrdering(outboxMessages, expected);
+  });
+
+  it('go to section completed', async () => {
```

Comment:
Is this necessary if the next step is to log in as a different user?

Reply:
Good catch.. not necessary at all..